### PR TITLE
SqlNullable is supported by ConstraintEvaluation

### PR DESCRIPTION
**Pull request by jgaleotti**

### CODE REVIEW

File: dbconstraint/src/main/java/org/evomaster/dbconstraint/parser/jsql/JSqlVisitor.java [hunk 1]
```diff
@@ -123,8 +107,25 @@ public void visit(Function function) {
...
+            SqlLiteralValue negated;
+            if (sqlLiteralValue instanceof SqlBigIntegerLiteralValue) {
+                SqlBigIntegerLiteralValue sqlBigIntegerLiteralValue = (SqlBigIntegerLiteralValue) sqlLiteralValue;
+                negated = new SqlBigIntegerLiteralValue(sqlBigIntegerLiteralValue.getBigInteger().negate());
+            } else if (sqlLiteralValue instanceof SqlBigDecimalLiteralValue) {
+                SqlBigDecimalLiteralValue sqlBigDecimalLiteralValue = (SqlBigDecimalLiteralValue) sqlLiteralValue;
+                negated = new SqlBigDecimalLiteralValue(sqlBigDecimalLiteralValue.getBigDecimal().negate());
+
+            } else {
+                throw new RuntimeException("Extraction of condition not yet implemented");
```
Comment (arcuri82) [inline dbconstraint/src/main/java/org/evomaster/dbconstraint/parser/jsql/JSqlVisitor.java]: could be useful to add `sqlLiteralValue .getClass` to the error message

File: core/src/main/kotlin/org/evomaster/core/database/DbActionGeneBuilder.kt [hunk 2]
```diff
@@ -160,7 +155,11 @@ class DbActionGeneBuilder {
 
     private fun handleBigIntColumn(column: Column): Gene {
         return if (column.enumValuesAsStrings != null) {
-            EnumGene(column.name, column.enumValuesAsStrings.map { it.toLong() })
+            if (column.enumValuesAsStrings.isEmpty()) {
+                throw IllegalArgumentException("the list of enumerated values cannot be empty")
+            } else {
```
[the remark this answers is not in the document]
Comment (jgaleotti) [reply core/src/main/kotlin/org/evomaster/core/database/DbActionGeneBuilder.kt]: ok.

File: dbconstraint/src/main/java/org/evomaster/dbconstraint/parser/jsql/JSqlVisitor.java (hunk 1 above)
Comment (jgaleotti) [reply dbconstraint/src/main/java/org/evomaster/dbconstraint/parser/jsql/JSqlVisitor.java]: thanks

File: core/src/main/kotlin/org/evomaster/core/database/SqlInsertBuilder.kt [hunk 3]
```diff
@@ -73,13 +71,87 @@ class SqlInsertBuilder(
                     throw IllegalArgumentException("Column in different table: ${c.table}!=${t.name}")
                 }
 
+                var lowerBound: Int?
+                run {
```
[the remark this answers is not in the document]
Comment (jgaleotti) [reply core/src/main/kotlin/org/evomaster/core/database/SqlInsertBuilder.kt]: I tried to used a single block {...} to fix the scope of some values. It seems as Kotlin does not allow single blocks without run {...} command. Nevertheless, I refactored the code to make it more readable

**Pull request merged**